### PR TITLE
(#5) - read live data from cloudant and filter out old data format

### PR DESCRIPTION
**Pull request by vw52**

### CODE REVIEW

File: dashboard.js [hunk 1]
```diff
@@ -45,34 +45,45 @@
...
+  
+    var data = new Array();
+    for (var i = 0; i < docs.rows.length; i++) {
+      if (docs.rows[i].doc.branch === 'master' &&
+          !docs.rows[i].doc.pull_request) {
+        data.push(docs.rows[i]);
+      }
+    }
+    
+    /*data.sort(function (a, b) {
```
Comment (daleharvey) [inline dashboard.js]: Just delete code you dont need anymore

File: dashboard.js [hunk 2]
```diff
@@ -45,34 +45,45 @@
...
       "bulk-inserts"
     ];
-  
     color.domain(varNames);
   
     var resultsData = varNames.map(function (name) {
       return {
         name: name,
         durations: data.map(function (d) {
+          console.log(d);
```
Comment (daleharvey) [inline dashboard.js:70]: Same here

File: dashboard.js [hunk 3]
```diff
@@ -45,34 +45,45 @@
...
-    });
-  
+  //read live data from cloudant
+  var db = new PouchDB('http://pouchdb.cloudant.com/performance_results');
+  db.allDocs({"include_docs": true}).then(function(docs) {
+  
+    var data = new Array();
+    for (var i = 0; i < docs.rows.length; i++) {
+      if (docs.rows[i].doc.branch === 'master' &&
+          !docs.rows[i].doc.pull_request) {
```
Comment (daleharvey) [inline dashboard.js]: so this test is slightly wrong, it should be 

```
if (docs.rows[i].doc.branch === 'master' &&
    docs.rows[i].doc.pull_request === 'false') { 
```

**Comment by daleharvey** [comment]
So theres only one data point when showing this, I think its because of the test I mentioned

Few nits about uneeded code, but apart from that it looks great, thanks, if we can gix it to show more than one data point then looks ready to merge

**Comment by virgie** [comment]
The reason it shows only one data point is because of the incorrect test in lines 54 and 55, as pointed out.

Also, the line 33 needs to be changed to this in order to get the lines properly drawn:

```
return x((new Date(d.date)).getTime()) + x.rangeBand() / 2;
```

This change is needed because d.date isn't a number anymore.

**Comment by vw52** [comment]
Thanks Dale and Virginia. I think I addressed all of the above now. 

The x-axis isn't readable with this many data points.

**Comment by virgie** [comment]
We can open up another issue for that after this gets merged

**Comment by virgie** [comment]
I've fixed the x-axis so once this gets merged, I can make a pull request with my changes

**Comment by daleharvey** [comment]
Awesome, merged the cloudant patch, checking out the axis changes in the other PR, this is sweet

https://github.com/pouchdb/dashboard/commit/1cc2140dcaf3d2591211a3f62836c0d225f9d05f

**Pull request closed without merging**